modules/storages/app/views/storages/admin/storages/edit.html.erb

```diff
@@ -28,11 +56,11 @@
         <div class="attributes-key-value--key"><%= t("storages.provider_types.#{@object.provider_type}") %> <%= t(:'storages.label_oauth_client_secret') %></div>
         <div class="attributes-key-value--value-container">
           <div class="attributes-key-value--value -text">
-            <span><%= @short_secret %></span>
+            <span><%= short_secret(@object.oauth_client.client_secret) %></span>
```

Comment:
this is not what UX decided to do here. It looks weird of shortening one secret and hiding the other. We should be consistent and consistent with OP UI is hiding secrets. Why should we reveal any of the characters?